scripts/release/createPWTRelease.ts

```diff
@@ -44,24 +45,22 @@ async function main(): Promise<void> {
     }
 
     // 4. Do release
-    console.log("Starting pluggable-widget-tools release...");
+    console.log("Preparing pluggable-widget-tools release...");
 
     const remoteName = `origin-${packageInfo.packageName}-v${packageInfo.version.format()}-${makeid()}`;
```

Comment:
```suggestion
    const remoteName = `origin-${packageInfo.packageName}-v${packageInfo.version.format()}-${Date.now()}`;
```
As Date.now() always give different results I think this will serve it's purpose + we can get rid of `makeid`